js/members.js

```diff
@@ -83,3 +135,12 @@ function clickThirdYears() {
 function clickSecondYears() {
     $('#allMembersContainer').html(members['secondYears']);
 }
+
+function clickAllAlumni() {
+    $('#allMembersContainer').html(members['firstAlumnis']);
+    //hide present members batchButton
+    $("#batchButtons button").css("display", "none");
```

Comment:
Same as above (set ```display: none``` on the container div itself instead of on every single child button)

Reply:
done